.github/workflows/daily.yml

```diff
@@ -15,10 +15,7 @@ jobs:
         server:
           [
             as-hk-1,
-            eu-fin-1,
             eu-fin-2,
-            eu-ger-1,
-            eu-ger-2,
```

Comment:
We need to add `as-hk-1` here. I tried to add a suggestion but GitHub is garbage and won't let me copy-paste. I refuse to count how many spaces are there and enter them manually.

Reply:
Probably not haha. I'll have it pull from a list once that's available.

Reply:
> We need to add as-hk-1 here.

Already in the list, so I approve your approval.